### PR TITLE
chore: move explorer to devdependency [4.x]

### PR DESCRIPTION
**Pull request by jannyHou**
Update for 4.x version. Same as https://github.com/strongloop/loopback-workspace/pull/567

Move explorer from dependency to devDependency to prevent the snyk warning.
See https://github.ibm.com/velox/support/issues/668#issuecomment-23073287

The node 6 failure shouldn't be relevant.

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-workspace) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](https://loopback.io/doc/en/contrib/style-guide-es6.html)
- [ ] Commit messages are following our [guidelines](https://loopback.io/doc/en/contrib/git-commit-messages.html)

### CODE REVIEW

File: package.json
Comment (dhmlau) [inline package.json:3]: when we publish a release, does it modify the version automatically?

Comment (jannyHou) [reply package.json:3]: right...I created a branch 4.x_latest for publishing the 4.x patches. When publishing there the version should change. Let me revert this.

**Review by hacksparrow — APPROVED** [review]

**Review by bajtos — CHANGES_REQUESTED** [review]
Let's wait until https://github.com/strongloop/loopback-workspace/pull/567 is landed first, and the back-port the final commit.

Also the branch containing code for `4.x.y` versions should be called `4.x`, see https://github.com/strongloop/loopback/tree/2.x for an example.

The branches with `latest` suffix are (use to be?) created by `slt-release` tooling.

**Comment by bajtos** [comment]
> Also the branch containing code for `4.x.y` versions should be called `4.x`, see https://github.com/strongloop/loopback/tree/2.x for an example.
> 
> The branches with `latest` suffix are (use to be?) created by `slt-release` tooling.

Uh oh, I changed the target branch of this pull request from `4.x_latest` to `4.x` and now there is a bunch of commits in this patch.

Can you @jannyHou please run `git rebase -i 4.x` and clean up the commit history? I am afraid there are merge conflicts that I don't have bandwidth to resolve myself.

It may be better to wait until  #567 is landed first, then start from with a fresh plate by running `git reset --hard origin/4.x`, and finally cherry-pick the final commit(s) from #567.

**Comment by jannyHou** [comment]
@bajtos I checked branch 4.x but it doesn't contain all the 4.x commits, it's quite out of date...If you look at the commit history, the 2nd page:
https://github.com/strongloop/loopback-workspace/commits/master?after=8abfd2d9e0921a9926b26301e1c27fea589c5adb+34&branch=master

The latest 4.x release is 4.5.2 (Commits on Jan 28, 2019), but branch 4.x only has commits in 2017...that's why I created another branch to track version 4.

**Review by dhmlau — APPROVED** [review]
LGTM. Changes are the same as the PR #567 for the master branch.

**Pull request merged**
